javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -22,15 +24,18 @@
  * @author Ruslan Sennov
  * @since 2.0.0
  */
-public final class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T>, Serializable {
-
+public final class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T> {
```

Comment:
It would be better to explicitly state that Vector is Serializable by declaring the interface in the implements section. I've checked the other collection types and we did it not consequently. But I think we should do it for all collections (in a separate PR), it is more robust regarding changes of super-interfaces.